HW_5/task_3.py

```diff
@@ -0,0 +1,48 @@
+from random import randint
+import os
+
+
+def print_board(array: list):
+    print('|', end='')
```

Comment:
Немного съехало отображение
![image](https://user-images.githubusercontent.com/49741194/207256712-331342e6-c664-4fb0-8495-302dcb024cc4.png)
```suggestion
    print('| ', end='')
```